packages/beacon-node/src/api/rest/swaggerUI.ts

```diff
@@ -0,0 +1,32 @@
+import {FastifyInstance} from "fastify";
+import {BeaconRestApiServerOpts} from "./index.js";
+
+export async function registerSwaggerUIRoutes(
+  server: FastifyInstance,
+  opts: BeaconRestApiServerOpts,
+  version = ""
+): Promise<void> {
+  await server.register(await import("@fastify/swagger"), {
+    openapi: {
+      info: {
+        title: "Lodestar API",
+        description: "",
```

Comment:
Suggestions welcome here.

For an example of what can be added and what it renders as, check out:

https://ethereum.github.io/beacon-APIs/releases/v2.4.2/beacon-node-oapi.json

https://ethereum.github.io/beacon-APIs/#/